docs/dashboards/README.md

```diff
@@ -0,0 +1,78 @@
+
+The dashboards use metrics from the collectors' [internal telemetry](https://opentelemetry.io/docs/collector/internal-telemetry/).
+See the [list of internal metrics](https://opentelemetry.io/docs/collector/internal-telemetry/#lists-of-internal-metrics) for an overview of which metrics are available.
+
+## Prerequisites
+The dashboards rely on the self-monitoring capabilities of the OTel collector as well as certain attributes on the exported metrics data.
+Required attributes are: 
+- `service.name` (automatically added by the collector and added to data ingested by Dynatrace)
+- `service.instance.id` (automatically added by the collector, needs to be [added to the Dynatrace attribute allow list](#adding-serviceinstanceid-to-the-allow-list))
+
+Dynatrace accepts metrics data with Delta temporality via OTLP/HTTP.
+Collector and Collector Contrib versions 0.107.0 and above as well as Dynatrace collector versions 0.12.0 and above support exporting metrics data in that format.
+Earlier versions ignore the `temporality_preference` flag and would, therefore, require additional processing (cumulative to delta conversion) before ingestion.
+It is possible to to this conversion in a collector, but would make the setup more complicated, so it is initially omitted in this document.
+
+### Adding `service.instance.id` to the allow list
+While `service.name` is on the Dynatrace OTLP metrics ingest allow list by default, `service.instance.id` is not.
+To add it, follow [this guide](https://docs.dynatrace.com/docs/shortlink/metrics-configuration#allow-list) and add `service.instance.id` to the list.
+This will ensure that this resource attribute is stored as a dimension on the metrics in Dynatrace. 
+
+## Sending internal telemetry to Dynatrace
+Every OpenTelemetry collector has self-monitoring capabilities, but they need to be activated.
+Self-monitoring data can be exported from the collector via the OTLP protocol.
+The configuration below assumes the environment variables `DT_ENDPOINT` and `DT_API_TOKEN` to be set.
+In order to send data to Dynatrace via OTLP, you will need to supply a Dynatrace endpoint and an ingest token with the `metrics.ingest` scope set.
+See the [Dynatrace docs](https://docs.dynatrace.com/docs/extend-dynatrace/opentelemetry/getting-started/otlp-export) for more information.
+
+To send self-monitoring data to Dynatrace, use the following configuration:
+
+```yaml
+service:
+  # turn on selfmon
+  telemetry:
+    metrics:
+      # metrics verbosity level. Higher verbosity means more metrics. 
+      # The dashboard relies on metrics at level detailed.
+      level: detailed
+      # set up OTLP exporter
+      readers:
+        - periodic:
+            interval: 60000
+            exporter:
+              otlp:
+                protocol: http/protobuf
+                temporality_preference: delta
+                endpoint: "${env:DT_ENDPOINT}/api/v2/otlp/v1/metrics"
```

Comment:
This should be aligned with other examples in the docs.
```suggestion
                endpoint: "${env:DT_ENDPOINT}"
```

Reply:
In 13552d1320f1fa05636380e8f4ee6ed245fcc3f4, I changed it to use DT_URL instead of DT_ENDPOINT. That way, if we start exporting traces we can reuse the ENV VAR instead of having to add a new one just for traces.